your_make.sample

```diff
@@ -1,4 +1,32 @@
-## --------------- SAMPLE TARGET CHAIN -------------------
+## -------------- EXPORT BLOCK ------------------------
+
+# configuration file
+file := ~/config.inc.sh
+# Add an export here for each variable from the configuration file that you are going to use in the targets.
+export USER_NAME = $(shell sed -n -e '/^USER_NAME/p' ${file} | cut -d "=" -f 2)
+export SLACK_CHANNEL = $(shell sed -n -e '/^SLACK_CHANNEL/p' ${file} | cut -d "=" -f 2)
+export SLACK_BOT_NAME = $(shell sed -n -e '/^SLACK_BOT_NAME/p' ${file} | cut -d "=" -f 2)
+export SLACK_BOT_EMOJI = $(shell sed -n -e '/^SLACK_BOT_EMOJI/p' ${file} | cut -d "=" -f 2)
+export SLACK_BOT_KEY = $(shell sed -n -e '/^SLACK_BOT_KEY/p' ${file} | cut -d "=" -f 2)
+
+# these makefiles stored in geocint-runner and geocint-openstreetmap repositories
+# runner_make contains basic set of targets for creation project folder structure
+# osm_make contains contain set of targets for osm data processing
+include runner_make osm_make
+
+## ------------- CONTROL BLOCK -------------------------
+
+# replace your_final_target placeholder with the names of final target, that you will use to run pipeline
+# you can also add here the names of targets that should not be rebuilt automatically, but only when conditions are met or at your request
```

Comment:
please, add an example for unexperienced with geocint users. on how it should look like

Reply:
added explanation